packages/fluent-theme/src/components/preChatActivity/StarterPromptsToolbar.tsx

```diff
@@ -1,29 +1,39 @@
+import { hooks } from 'botframework-webchat-api';
 import { type DirectLineCardAction } from 'botframework-webchat-core';
 import cx from 'classnames';
-import React, { memo, type ReactNode } from 'react';
+import React, { Fragment, memo } from 'react';
 import { useStyles } from '../../styles/index.js';
 import StarterPromptsCardAction from './StarterPromptsCardAction.js';
 import styles from './StarterPromptsToolbar.module.css';
 
+const { useUIState } = hooks;
+
 type Props = Readonly<{
   cardActions: readonly DirectLineCardAction[];
   className?: string | undefined;
-  children?: ReactNode | undefined;
 }>;
 
-const StarterPrompts = ({ cardActions, children, className }: Props) => {
+const StarterPrompts = ({ cardActions, className }: Props) => {
   const classNames = useStyles(styles);
+  const [uiState] = useUIState();
 
   return (
     // TODO: Accessibility-wise, this should be role="toolbar" with keyboard navigation.
     <div className={cx(className, classNames['pre-chat-message-activity__card-action-toolbar'])}>
       <div className={classNames['pre-chat-message-activity__card-action-toolbar-grid']}>
-        {children ||
+        {uiState === 'blueprint' ? (
+          <Fragment>
+            <StarterPromptsCardAction />
+            <StarterPromptsCardAction />
+            <StarterPromptsCardAction />
+          </Fragment>
+        ) : (
           cardActions
             .filter<
               DirectLineCardAction & { type: 'messageBack' }
             >((card: DirectLineCardAction): card is DirectLineCardAction & { type: 'messageBack' } => card.type === 'messageBack')
-            .map(cardAction => <StarterPromptsCardAction key={cardAction.text} messageBackAction={cardAction} />)}
+            .map(cardAction => <StarterPromptsCardAction key={cardAction.text} messageBackAction={cardAction} />)
```

Comment:
nit: same text would trigger warning

Reply:
Nothing would be useful other than `index`. 🤣